django/db/models/fields/related.py

```diff
@@ -1866,8 +1880,8 @@ class ForeignKey(ForeignObject):
     }
     description = _("Foreign Key (type determined by related field)")
 
-    def __init__(self, to, to_field=None, related_name=None, related_query_name=None,
-            limit_choices_to=None, parent_link=False, on_delete=CASCADE,
+    def __init__(self, to, on_delete, related_name=None, related_query_name=None,
```

Comment:
Argh, of course that's true. My fault, I wasn't thinking clearly. Commented on a line below...